css/historical-navigation.less

```diff
@@ -35,6 +35,11 @@
                 }
             }
 
+            .historical-navigation-current {
+                white-space: nowrap;
```

Comment:
I think this'll mean the issue Ben saw will happen with this too: https://boomii.atlassian.net/browse/FLOW-1939
(It doesn't wrap and causes the name to go off the edge of the screen)